Sources/SwiftParser/Parser.swift

```diff
@@ -101,6 +101,15 @@ public struct Parser {
   /// When this nesting level is exceeded, the parser should stop parsing.
   let maximumNestingLevel: Int
 
+  /// See commens in ``IncrementalParseLookup``
+  var parseLookup: IncrementalParseLookup?
+
+  /// See comments in ``LookaheadRanges``
+  public var lookaheadRanges = LookaheadRanges()
```

Comment:
Ah I just realized that actually there’s no reason to make it publicly writable. Not sure if 

```suggestion
  public internal(set) var lookaheadRanges = LookaheadRanges()
```